docs/data/date-pickers/base-concepts/ReferenceDateExplicitDateTimePicker.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+
+export default function ReferenceDateExplicitDateTimePicker() {
+  const [value, setValue] = React.useState<Dayjs | null>(null);
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <Stack spacing={2} sx={{ minWidth: 270 }}>
```

Comment:
Nitpick: To avoid the layout shift after a value is selected.
```suggestion
      <Stack spacing={2} sx={{ minWidth: 305 }}>
```